hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/ReadableUpgradeStoreImpl.java

```diff
@@ -40,10 +42,12 @@
  * <p>This class is not exported from the module. It is an internal implementation detail.
  */
 public class ReadableUpgradeStoreImpl implements ReadableUpgradeStore {
+    private final FileID UPGRADE_FILE_ID = new FileID(0, 0, 150);
```

Comment:
We should bring up with the team (perhaps next week) how to get configuration into object store implementations.

Reply:
sure will be done. we can also check with @netopyr